JS-Algorithms/challenges.js

```diff
@@ -41,7 +65,11 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  let circularIndex = (index % COUNTRY_NAMES.length === 0) ? 0 : index % COUNTRY_NAMES.length;
```

Comment:
we can change it to only be `let circularIndex = index % COUNTRY_NAMES.length`. With that change the values will be from 0 to 4 which are the indexes of the country names array